src/main/resources/templates/db_scripts/create-table.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE tags
+DROP COLUMN jsonb,
+ADD COLUMN record_id VARCHAR (50) NOT NULL,
+ADD COLUMN record_type VARCHAR (50) NOT NULL,
+ADD COLUMN tag VARCHAR (50) NOT NULL;
+
+INSERT INTO tags (record_id, record_type, tag) VALUES
+('583-4345-762169', 'resource', 'first tag'),
+('4345', 'package', 'second tag'),
+('762169', 'title', 'third tag');
```

Comment:
Looks good Natalia
Just a few questions for understanding:
- assuming tables will be tenant specific - correct assumption?
- will we have a predefined set of record types (provider, package, resource and tile) for example?
- trying to understand if our package (record_id) would be of the format vendorId-packageId or package_id? -- I think vendorId-packageId will give us a unique package

Reply:
> assuming tables will be tenant specific - correct assumption?

the name of the table will be 'tags' but the name of the schema will consist of the following parts: `<tenant>_<moduleName>`
as an example, please see the following image
<img width="300" alt="screen shot 2019-02-04 at 11 21 42 am 1" src="https://user-images.githubusercontent.com/37537790/52199571-2e9c8000-286f-11e9-82d1-80198c85e621.png">

> will we have a predefined set of record types (provider, package, resource and tile) for example?

yes, we will have test data . The code for inserting data will be stored in separate file
`INSERT INTO tags (record_id, record_type, tag) VALUES
('583-4345-762169', 'resource', 'first tag'),
('4345', 'package', 'second tag'),
('762169', 'title', 'third tag');`
I will add possibility to add test data by environment flag in nex commit

> trying to understand if our package (record_id) would be of the format vendorId-packageId or package_id? -- I think vendorId-packageId will give us a unique package

here we can decide what is more convinient for us in case of search

Reply:
It will be the same value that comes from UI: vendorId-packageId

Reply:
Thanks for all of these details and explanation!